manifests/cloud-platform-alt-dns/coredns.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     app: cloud-platform-infra-coredns
   annotations:
     target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+    openshift.io/required-scc: privileged
```

Comment:
@cybertron removed it long time ago for a reason. Why do we need it? It's just a DNS container, I can't see an obvious reason for it to be privileged, but I may be missing something subtle

Reply:
@mko, @cybertron removed setting to run the CoreDNS pod's containers in privileged mode with https://github.com/openshift/machine-config-operator/pull/4443.

Change above is adding work done in https://github.com/openshift/machine-config-operator/pull/4504.